model_compression_toolkit/core/keras/kpi_data_facade.py

```diff
@@ -15,15 +15,13 @@
 
 from typing import Callable
 
-from model_compression_toolkit.core import MixedPrecisionQuantizationConfig, CoreConfig, MixedPrecisionQuantizationConfigV2
+from model_compression_toolkit.core import CoreConfig, MixedPrecisionQuantizationConfigV2
```

Comment:
Isn't MixedPrecisionQuantizationConfigV2 going to be removed in the new API (removing the V2)?
Is it a part of a different PR?

Reply:
It's in a different PR.